cli/cmd/root.go

```diff
@@ -24,6 +24,9 @@ type Config struct {
 }
 
 var Retina = &cobra.Command{
+	Use: "kubectl-retina",
```

Comment:
`completion` command retrieves the tool name from the first word of `Use` field.